profiler/src/main/java/com/splunk/opentelemetry/profiler/TLABProcessor.java

```diff
@@ -86,13 +88,26 @@ public void accept(RecordedEvent event) {
         LogDataBuilder.create(resource, INSTRUMENTATION_LIBRARY_INFO)
             .setName(PROFILING_SOURCE)
             .setEpoch(time)
-            .setBody(stack)
+            .setBody(threadLine + "\n" + "   java.lang.Thread.State: UNKNOWN\n" + stack)
```

Comment:
WDYT about moving this concatenation to `buildThreadLine()` (possibly renaming it) to avoid using that additional `StringBuilder`?

Reply:
Sure, lemme have a look.

Reply:
ok, tried that out. LMK what you think.